GUIShop/shops.yml

```diff
@@ -489,6 +504,10 @@ Gear:
     id: SADDLE
     buy-price: 75.0
     sell-price: 30.0
+  '27':
+    type: SHOP
+    id: SHIELD
+    buy-price: 100.0

```

Comment:
Maybe reduce the price to 50, this way people can buy one just after they've joined

Reply:
50 is ok